app/imports/ui/components/ProfileSkillsDropdown.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Dropdown, Header, Loader, Container } from 'semantic-ui-react'
+import PropTypes from 'prop-types';
+import { withTracker } from 'meteor/react-meteor-data';
+import { Meteor } from "meteor/meteor";
+import { Skills } from '../../api/skill/skill';
+
+class ProfileSkillsDropdown extends React.Component {
+  render() {
+    return (this.props.ready) ? this.renderDropdown() : <Loader active>Retrieving data</Loader>;
+  }
+
+
+  skillOptions = this.props.skills.map((skill) => ({
+    key: skill._id,
+    text: skill.name,
+    value: skill.name,
+    description: skill.description,
+  }));
+
+
+  // render() {
+  //     if (this.props.ready) {
+  //       console.log('Got Skills');
+  //       return this.renderDropdown();
+  //     } else {
+  //       console.log('Retrieving Skills');
+  //       return <Loader active>Getting data</Loader>;
+  //     }
+  // }
+
+  renderDropdown() {
+    return (
+      <Container>
+
+
+        <Dropdown
+          placeholder='Skill'
+          fluid
+          multiple
+          search
+          selection
+          options={this.skillOptions}
+        />
+      </Container>
+    )
+  }
+}
+
+ProfileSkillsDropdown.propTypes = {
+  skills: PropTypes.array.isRequired,
+  // currProfile: PropTypes.object,
```

Comment:
did you want to keep this?

Reply:
yeah if I'm going to keep working on the skills (specifically, linking the skills to the student and vice versa).  If i'm not going to work on skills anymore i can delete it